web/src/components/troubleshoot/SupportBundleAnalysis.jsx

```diff
@@ -213,11 +213,12 @@ export class SupportBundleAnalysis extends React.Component {
     this.getSupportBundle();
   }
   componentWillUnmount() {
-    this.state.pollForBundleAnalysisProgress.stop();
+    clearInterval(this.pollingRef.current);
```

Comment:
this is good. makes more sense to put the callback ref into the state rather than an object with callbacks on it